### DISCUSSION TITLE
there is no dots or button showing up ?

### DISCUSSION THREAD

**Discussion by Adamkaram** [davidjerleke/embla-carousel#553] [Help] [answered]
i'm new to embla and i'm scoked it's wonderful but for bad i can't control the apperence of dots and button for the slide 

i'm generating this example and it's quite good 
https://codesandbox.io/s/embla-carousel-arrow-dots-react-forked-mz4ff9?file=/src/css/embla.css:0-2618
![image](https://github.com/davidjerleke/embla-carousel/assets/52092726/c3afac44-bd7f-48cf-9da9-6e158ac7d8a0)

in my nextjs project this what i got 
![image](https://github.com/davidjerleke/embla-carousel/assets/52092726/b68cb132-5d76-4eb7-8c6d-02662f25e708)

how i can make these dots and button working just as fine

**Comment by davidjerleke** [comment, marked as answer]
Hi @Adamkaram,

Thank you for your question. As you can see on the sandbox link you shared, everything is working as expected and the buttons are there. This is most likely caused because you've missed to copy some parts of the code from the sandbox to your project. If you don't get any script errors, you're probably missing some CSS. Please triple check that you've copied all code from the sandbox to your project and use Chrome Dev Tools to find out what's missing.

Best,
David
